modules/virt-enabling-descheduler-evictions.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// virt/virtual_machines/advanced_vm_management/virt-enabling-descheduler-evictions.adoc
+
+:_content-type: PROCEDURE
+[id="virt-enabling-descheduler-evictions_{context}"]
+= Enabling descheduler evictions on a VM
+
+After the descheduler is installed, you can enable descheduler evictions on your VM with an annotation to the custom resource (CR).
+
+.Procedure
+
+. Install the descheduler in the {product-title} web console or OpenShift CLI (`oc`).
+. Prior to starting the VM, add the `descheduler.alpha.kubernetes.io/evict` annotation to the VM CR:
```

Comment:
The first part sounds like another prerequisite - "The virtual machine must not be running"